packages/notifications/src/styled/global-alert/StyledGlobalAlertButton.ts

```diff
@@ -6,65 +6,98 @@
  */
 
 import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
-import { getColorV8, DEFAULT_THEME, focusStyles } from '@zendeskgarden/react-theming';
+import {
+  getColor,
+  DEFAULT_THEME,
+  focusStyles,
+  retrieveComponentStyles,
+  ColorParameters
+} from '@zendeskgarden/react-theming';
 import { Button } from '@zendeskgarden/react-buttons';
 
 import { IGlobalAlertProps } from '../../types';
 import { colorStyles as basicColorStyles } from './StyledGlobalAlertClose';
 
+const COMPONENT_ID = 'notifications.global_alert.button';
+
 interface IStyledGlobalAlertButtonProps {
-  alertType: IGlobalAlertProps['type'];
+  $alertType: IGlobalAlertProps['type'];
   isBasic?: boolean;
 }
 
-function colorStyles(props: ThemeProps<DefaultTheme> & IStyledGlobalAlertButtonProps) {
-  if (props.isBasic) {
+type OffsetOptions = Record<string, Record<string, ColorParameters['offset']>>;
+
+function colorStyles(
+  props: IStyledGlobalAlertButtonProps & ThemeProps<DefaultTheme> & IStyledGlobalAlertButtonProps
+) {
+  const { $alertType, isBasic, theme } = props;
+
+  if (isBasic) {
     return basicColorStyles(props);
   }
 
-  let backgroundColor;
-  let hoverBackgroundColor;
-  let activeBackgroundColor;
-  let focusColor;
+  let bgVariable;
+  let offsetOptions: OffsetOptions;
+  let offsetHoverOptions: OffsetOptions;
+  let offsetActiveOptions: OffsetOptions;
```

Comment:
```suggestion
  let offset: number;
  let offsetHover: number;
  let offsetActive: number;
```

...again, I think just using the `getColor({ theme, offset: <number> })` is the better way to go, and easier to read/maintain.